src/wizards/audience/views/setup/setup.js

```diff
@@ -179,212 +179,214 @@ export default withWizardScreen(
 						skipPrerequisite={ skipPrerequisite }
 					/>
 				) ) }
-			{ config.enabled && (
-				<Card noBorder>
-					<hr />
-					<SectionHeader
-						title={ __(
-							'Newsletter Subscription Lists',
-							'newspack-plugin'
-						) }
-					/>
-					<ActionCard
-						title={ __(
-							'Present newsletter signup after checkout and registration',
-							'newspack-plugin'
-						) }
-						description={ __(
-							'Ask readers to sign up for newsletters after creating an account or completing a purchase.',
-							'newspack-plugin'
-						) }
-						toggleChecked={ config.use_custom_lists }
-						toggleOnChange={ value =>
-							updateConfig( 'use_custom_lists', value )
-						}
-					/>
-					{ config.use_custom_lists && (
-						<SortableNewsletterListControl
-							lists={
-								newspackAudience.available_newsletter_lists
-							}
-							selected={ config.newsletter_lists }
-							onChange={ selected =>
-								updateConfig( 'newsletter_lists', selected )
+			<Card noBorder>
```

Comment:
Nonblocking nit: It's possible for this `Card` component to have no children if `config.enabled` and `can_use_salesforce` are both false. Maybe we can nest each of these sections within their own `Card` components to avoid this case.

Reply:
That looks a lot better! Thank you for the suggestion.

Updated in 00e18fb803a79067520f0cfe9379e6d476dd13e7